code/lib/core-server/src/utils/__tests__/index-extraction.test.ts

```diff
@@ -604,53 +604,4 @@ describe('docs entries from story extraction', () => {
       }
     `);
   });
-  it(`Only adds a docs entry and not a story entry when an input has the "docsOnly" tag`, async () => {
```

Comment:
@shilman I need assistance here, I am not sure if we can just remove this test as it was targetting .stories.mdx, or if we need to test it differently with a new .mdx stub

Reply:
Get rid of it